src/Renderers/CodeNodeRenderer.php

```diff
@@ -73,6 +73,9 @@ public function render(): string
         if ('text' !== $language) {
             $highLighter = new Highlighter();
             $code = $highLighter->highlight(self::LANGUAGES_MAPPING[$language] ?? $language, $code)->value;
+
+            // this allows to highlight the $ in PHP variable names
+            $code = str_replace('<span class="hljs-variable">$', '<span class="hljs-variable"><span class="hljs-variable-other-marker">$</span>', $code);
```

Comment:
This is a hack (and I'm ok with that!). But we're going to need a test that shows that this transformation does happen. That way, when we upgrade highlightjs and they change something, we won't be surprised.